lib/MQTT/MqttClient.h

```diff
@@ -30,12 +30,14 @@
 #ifndef MQTTCLIENT_H
 #define MQTTCLIENT_H
 
-#include <PubSubClient.h>
-#include <Settings.h>
+#include <forward_list>
+
 #include <WString.h>
 #include <WiFiClient.h>
 
-class SetHandler;
+#include <Settings.h>
+
+class PubSubClient;
```

Comment:
why is this necessary?

Reply:
For some reason it didn't want to compile otherwise.